src/components/AssetInfo/TokenSideBarContent.test.tsx

```diff
@@ -1,20 +1,25 @@
 import React from "react";
 import { mount } from "enzyme";
 import TokenSideBarContent from "./TokenSideBarContent";
+import TokenSideBarHolder from "./TokenSideBarHolder";
+import TokenSideBarBeneficiary from "./TokenSideBarBeneficiary";
 
 describe("tokenSideBarContent", () => {
-  it("should have 2 fields for Holder with no endorsement awaiting", () => {
-    const wrapper = mount(<TokenSideBarContent isHolder={true} />);
-    expect(wrapper.find("section")).toHaveLength(2);
+  it("should render <TokenSideBarHolder /> view", () => {
```

Comment:
remove <> from component. can just write "should render TokenSideBarContent" component.